### PR TITLE
Update CGP13 with no longer proxying linked libraries

### PR DESCRIPTION
**Pull request by nambrot**
This PR updates CGP13 with the decision to no longer proxy linked libraries

### CODE REVIEW

File: CGPs/0013.md [hunk 1]
```diff
@@ -34,18 +34,14 @@ This Celo Core Contracts Upgrade proposal is special since it operationalizes th
...
-2. Registry `setAddressFor` AddressLinkedList library
-3. Registry `setAddressFor` AddressSortedLinkedList library
-4. Registry `setAddressFor` AddressSortedLinkedListWithMedian library
-5. Registry `setAddressFor` Proposals library
-6. Registry `setAddressFor` IntegerSortedLinkedList library
-
 Regular features that were being added or bugs that were fixed are described in the [Github release](https://github.com/celo-org/celo-monorepo/releases/tag/celo-contracts-v1.rc1). Since the major version of these Celo Core Contracts changed due to the version change, no separate transactions will reflect the "regular" changes. One exception to that is the newly added `DowntimeSlasher`:
 
 1. Registry `setAddressFor` DowntimeSlasher
+2. DowntimeSlasherProxy `setAndInitializeImplementation`:
```
Comment (ewilz) [inline CGPs/0013.md:40]: is this relevant to not proxying libraries?

**Review by yorhodes — APPROVED** [review]
LGTM

File: CGPs/0013.md (hunk 1 above)
Comment (nambrot) [reply CGPs/0013.md:40]: Not quite, its part of us moving initialization into the proposal itself
